apps/OboeTester/app/src/main/res/layout/activity_dynamic_workload.xml

```diff
@@ -38,7 +38,7 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_marginRight="8sp"
-            android:text="Perf Hint" />
+            android:text="ADPF" />
```

Comment:
Do we have a description of ADPF anywhere on the test?

Reply:
I added some comments at the top of the file.